### PR TITLE
Beemis: Curse of God

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Beemis: Curse of God

## Game Information
- **Title**: Beemis: Curse of God
- **URL**: https://blebgo.itch.io/beemis-the-curse-of-god

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/beemisthecurseofgod/Beemis - The Curse of God.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+# device_info.txt will be included by default
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+export PORT_32BIT="Y"
```
Comment (kloptops) [inline ports/beemisthecurseofgod/Beemis - The Curse of God.sh:21]: This needs to be before the `source mod_${CFW_NAME}` thingo above.

File: ports/beemiscurseofgod/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Beemis - The Curse of God.sh</path>
+    <name>Beemis: The Curse of God</name>
+    <desc>Become a real life bee and also fight god himself. Take on many bosses and wacky levels as a young bee whose life has been drastically changed by a curse maniacally concocted by God.</desc>
+    <releasedate>20180611T000000</releasedate>
+    <developer>blebuel</developer>
+    <publisher>Self-Published</publisher>
+    <genre>Action</genre>
+    <image>./beemiscurseofgod/screenshot.png</image>
```
Comment (JanTrueno) [inline ports/beemiscurseofgod/gameinfo.xml]: Should be ./beemisthecurseofgod/screenshot.png

**Review by JanTrueno — DISMISSED** [review]

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

File: ports/beemisthecurseofgod/Beemis - The Curse of God.sh (hunk 1 above)
Comment (tabreturn) [reply ports/beemisthecurseofgod/Beemis - The Curse of God.sh]: Thanks, done.

File: ports/beemiscurseofgod/gameinfo.xml (hunk 2 above)
Comment (tabreturn) [reply ports/beemiscurseofgod/gameinfo.xml]: Thanks, done.

**Review by kloptops — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**